.vale/fixtures/RedHat/ConfigMap/testinvalid.adoc

```diff
@@ -8,4 +8,4 @@ invalid content: config-map
 invalid content: config mail
 invalid content: config mapping
 invalid content: config abc
-invalid content: config
```

Comment:
config not followed by map is invalid content.

Reply:
The Supplementary Style Guide [entry for config map](https://redhat-documentation.github.io/supplementary-style-guide/#config-map) states that the following forms are valid:
- `[cC]onfig map`
- `ConfigMap` as an API object between backticks except in headings.
-- Can we scope this variation as valid for headings only and tell users to use backticks when it appears elsewhere?